### PR TITLE
Added HiddenFieldInspector (#78)

### LINKED ISSUE #78: Don't display fields of type HiddenField
Hi,   It seems that drf-yasg doesn't support hiding serializers.HiddenField's yet?  A hidden field in my serializer is showing up in the example data and model docs.

It looks like this is a known issue due to this TODO - https://github.com/axnsan12/drf-yasg/blob/9ad55bac9931a93ba2a9f7fd9f8693a0f8070c38/src/drf_yasg/inspectors/field.py#L472 but raising it.

### PR DESCRIPTION
**Pull request by therefromhere**
Attempt to resolve #78 , is this on the right track?  If so I'll look into how to add tests for this.

### CODE REVIEW

**Comment by axnsan12** [comment]
The implementation looks about right, but it could use some simple tests (basically just add a HiddenField somewhere in ``testproj`` and regenerate ``reference.yaml`` - https://github.com/axnsan12/drf-yasg/blob/master/CONTRIBUTING.rst#pull-requests).

The docs for ``DEFAULT_FIELD_INSPECTORS`` should also be updated to match its new reality.

File: src/drf_yasg/inspectors/field.py [hunk 1]
```diff
@@ -465,11 +465,21 @@ def field_to_swagger_object(self, field, swagger_object_type, use_references, **
         return NotHandled
 
 
+class HiddenFieldInspector(FieldInspector):
+    """Hide ``HiddenField``."""
+
+    def field_to_swagger_object(self, field, swagger_object_type, use_references, **kwargs):
+        if isinstance(field, serializers.HiddenField):
+            return None
+        else:
```
Comment (axnsan12) [inline src/drf_yasg/inspectors/field.py]: The ``else`` here is not needed, and could be removed to keep the style consistent with the other classes above.

**Comment by therefromhere** [comment]
done

**Comment by axnsan12** [comment]
Seems ok 👍, thanks!

**Pull request merged**